pkg/relay/relay.go

```diff
@@ -186,7 +186,9 @@ func (rs *Relay) GetHeader(ctx context.Context, m *structs.MetricGroup, request
 	header := maxProfitBlock.Header
 
 	if header.Header == nil || (header.Header.ParentHash != parentHash) {
-		logger.Debug(ErrBadHeader.Error())
+		if header.Header.ParentHash != parentHash {
+			logger.WithField("expected", parentHash).WithField("got", parentHash).Debug("invalid parentHash")
```

Comment:
I wonder shouldn't we just add this one for every log above.

Reply:
Not really, because we don't have `header.Header.ParentHash`, until we get it from the DB